Solutions/Threat Intelligence/Analytic Rules/IPEntity_DuoSecurity.yaml

```diff
@@ -50,5 +53,5 @@ entityMappings:
     fieldMappings:
```

Comment:
Mapping is following old procedure

Reply:
Updated the mapping, but since there is no data to test, not splitting Name to UPNSuffix.